### DISCUSSION TITLE
Undesired words delayed snatch

### DISCUSSION THREAD

**Discussion by papampi** [pymedusa/Medusa#9837] [Ideas & Feature requests]
We have the "Undesired words" in search setting which it says "results with words from this list will only be selected as a last resort" 
It would be nice to have a delay time settings to snatch the release with undesired words after that time.
I set x264 as undesired and x265 as preferred but x264 is usually available sooner than x265 so it snatch and download x264 and after an hour or so it snatches x265 and download it again.

**Comment by p0psicles** [comment]
Same has been requested as "upgrade based on preferred words" What, I think, comes down to the same

**Comment by papampi** [comment]
I mean medusa wait a set time before snatching a release with undesired word in it instead of snatching the undesired release and then upgrade it.

**Reply by p0psicles** [reply]
Maybe that's a bit to specific as a requirement. But I could translate it to something like. Wait x hours to start looking for a better quality and wait x hour to stop searching all together?

**Reply by papampi** [reply]
When it's possible to wait X hours to search for a better quality or wait for X hours to stop searching why not wait for X hours before snatch an undesired? 
Snatching and upgrading to a better quality or preferred release is great and it's good to snatch them as soon as they've been found. Only snatching undesired need to be delayed by X hours and snatch undesired if a preferred not found until then.

**Reply by p0psicles** [reply]
Well isn't undesired the opposite of preferred? I'm just looking for a solution that would also fix your problem scenario. And as I've already have done some work using preferred words. I'm looking for a solution that would use that. In steady of increasing complexity with the undesired words. 

But maybe I'm misunderstanding

**Comment by papampi** [comment]
Maybe I'm not eplaining quit well.
Usually 720p.x265 comes out a bit after 480p.x264 or 720p.x264, I added 720p.x265 to preferred, x264 to undesired and 720p.x264 to ignored so it won't satisfy the 720p preferred quality.
In past weeks Medusa keeps downloading 480p.x264 and replace them after an hour with the 720p.x265 one.
What I'm looking for is instead of first snatch x264 which is available sooner and then download the x265 a bit later, it waits a bit and directly snatch x265, if it couldn't snatch x265 in X hours then it can snatch x264 and continue looking for preferred one.

**Reply by p0psicles** [reply]
Can't you already do something like that with the search delay thats already available

**Reply by papampi** [reply]
Doesn't search delay available for providers?
How to do delay search for undesired words?

**Reply by p0psicles** [reply]
Yes, it just delays. But that kinda solves 90÷ of your issue right now. As in it will apply the undesired / preferred words tags, after x hours. 

Just thinking what might help you right now.

**Reply by papampi** [reply]
For now I removed SDTV from allowed qualities and added x264 to ignored words in search setting so it can only download 720p x265

**Reply by medariox** [reply]
I don't get it. Why don't you just set a 1 hour delay, so that both will be available and the x265 will be downloaded first?

**Reply by papampi** [reply]
Wow ...
Some times we are looking for a complex solution and don't see the simple one.
That's the easiest way and I never thought about it 🤦
